amivapi/tests/fixtures.py

```diff
@@ -258,12 +258,17 @@ def preprocess_events(self, schema, obj, fixture):
                 obj['time_register_end'] = (
                     datetime.utcnow() + timedelta(
                         seconds=random.randint(0, 1000000)))
+                obj['time_deregister_end'] = (
+                    datetime.utcnow() + timedelta(
+                        seconds=random.randint(0, 1000000)))
```

Comment:
time_deregister_end should be <= time_registter_end
Otherwise, people can still sign off after they can sign up which contradicts the use case for which we implement this feature.

Reply:
Also move deregister_end on top of register_end, as this deregister_end is (in general) earlier than register_end